builder/util/util.go

```diff
@@ -0,0 +1,83 @@
+package util
```

Comment:
This does not warrant an extra package. Just leave this is an extra file in the builder package.